content/docs/command-reference/diff.md

```diff
@@ -119,7 +119,7 @@ $ dvc diff
 
 ## Example: Comparing workspace with arbitrary commits
 
-<details>
+<details id="example-comparing-workspace-with-arbitrary-commits-expand">
```

Comment:
> Do you mean the name of the prop?

Actually, I meant these values for `ids`. It's better if @iterative/docs team can go through them once.

Reply:
For now they should be the same as now. In this case `click-and-expand-to-set-up-the-example` (link: https://dvc.org/doc/command-reference/diff#click-and-expand-to-set-up-the-example)

Reply:
```suggestion
<details id="example-arbitrary-commits-set-up-the-example">
```

Reply:
maybe remove example from prefix?
`<details id="example-arbitrary-commits-set-up-the-example">`
to
`<details id="arbitrary-commits-set-up-the-example">`

Reply:
Good call but from the end? `example-arbitrary-commits-set-up`

For consistency, since all my suggestions use prefixes.

Reply:
Good call but from the end? `example-arbitrary-commits-set-up`

For consistency, since all my suggestions use prefixes.

Reply:
UPDATE: I've edited all my similar suggestions removing `-the-example`